test/Validator/IdTest.php

```diff
@@ -45,9 +46,10 @@ public function testIsValidPhp71(int $bitsPerCharacter, string $id, bool $isVali
 
     public function testConstructorSetId(): void
```

Comment:
Testing constructors have set a property is a silly idea (Not yours, I know) - tests like this should be dropped in later PRs. This is fine for now.